include/portblas_helper.h

```diff
@@ -228,6 +228,23 @@ inline bool is_malloc_shared(sb_handle_t &sb_handle, const containerT _rs) {
   }
 }
 
+template <typename sb_handle_t>
+inline void check_intel_gpu_support(const sb_handle_t &sb_handle,
```

Comment:
The name of this function doesn't really match what it does. Perhaps `throw_unsupported_on_intelgpu`?

Reply:
Or be mores specific actually because iGPU is supported.

Reply:
Updated to `throw_unsupported_intel_dGPU` in [1d7efb9](https://github.com/codeplaysoftware/portBLAS/pull/524/commits/1d7efb99089ac2a85f20fc43790698ac77651f4a)